pypit/core/artracewave.py

```diff
@@ -226,6 +226,10 @@ def pad_dict(indict):
             if ampl[w[u]] > ampl[olduse][s]:
                 aduse[idxuse[s]] = False
                 break
+    # TODO Perhaps a more robust version of this code would only use the lines that were used in the wavelength solution. I guess
```

Comment:
Isn't that what this code does?

Reply:
You have special code in there to use the wv_calib only for the case where you want to remove ghosts. In general the code does not require  wv_calib. 

I guess the question is do we want to just require wv_calib. Or do we want to print out a warning, or better document tis. I can see some advantage to having this code work before a full wavelength solution exists, or if I want to run it on sky lines.